### PR TITLE
* #18: Fixed jackson-core/jackson-dataformat-cbor incompatible librar…

### LINKED ISSUE #18: Exception: Jackson jackson-core/jackson-dataformat-cbor incompatible library version detected
Running Exasol on AWS (version 7.0.3) and using the current release I get the following error message:

`
[Code: 0, SQL State: 22002]  VM error: F-UDF-CL-LIB-1127: F-UDF-CL-SL-JAVA-1002: F-UDF-CL-SL-JAVA-1013: 
com.exasol.ExaUDFException: F-UDF-CL-SL-JAVA-1080: Exception during run 
com.amazonaws.SdkClientException: Unable to marshall request to JSON: Jackson jackson-core/jackson-dataformat-cbor incompatible library version detected.
You have two possible resolutions:
		1) Ensure the com.fasterxml.jackson.core:jackson-core & com.fasterxml.jackson.dataformat:jackson-dataformat-cbor libraries on your classpath have the same version number
		2) Disable CBOR wire-protocol by passing the -Dcom.amazonaws.sdk.disableCbor property or setting the AWS_CBOR_DISABLE environment variable (warning this may affect performance)
com.amazonaws.services.kinesis.model.transform.DescribeStreamRequestProtocolMarshaller.marshall(DescribeStreamRequestProtocolMarshaller.java:59)
com.amazonaws.services.kinesis.AmazonKinesisClient.executeDescribeStream(AmazonKinesisClient.java:861)
com.amazonaws.services.kinesis.AmazonKinesisClient.describeStream(AmazonKinesisClient.java:846)
com.exasol.cloudetl.kinesis.KinesisShardsMetadataReader$.getAllShardsFromStream(KinesisShardsMetadataReader.scala:77)
com.exasol.cloudetl.kinesis.KinesisShardsMetadataReader$.run(KinesisShardsMetadataReader.scala:33)
com.exasol.cloudetl.kinesis.KinesisShardsMetadataReader.run(KinesisShardsMetadataReader.scala)
com.exasol.ExaWrapper.run(ExaWrapper.java:196)
Caused by: java.lang.RuntimeException: Jackson jackson-core/jackson-dataformat-cbor incompatible library version detected.
You have two possible resolutions:
		1) Ensure the com.fasterxml.jackson.core:jackson-core & com.fasterxml.jackson.dataformat:jackson-dataformat-cbor libraries on your classpath have the same version number
		2) Disable CBOR wire-protocol by passing the -Dcom.amazonaws.sdk.disableCbor property or setting the AWS_CBOR_DISABLE environment variable (warning this may affect performance)
com.amazonaws.protocol.json.SdkCborGenerator.getBytes(SdkCborGenerator.java:69)
com.amazonaws.protocol.json.internal.JsonProtocolMarshaller.finishMarshalling(JsonProtocolMarshaller.java:190)
com.amazonaws.protocol.json.internal.NullAsEmptyBodyProtocolRequestMarshaller.finishMarshalling(NullAsEmptyBodyProtocolRequestMarshaller.java:53)
com.amazonaws.services.kinesis.model.transform.DescribeStreamRequestProtocolMarshaller.marshall(DescribeStreamRequestProtocolMarshaller.java:57)
	... 10 more
Caused by: java.lang.NoSuchMethodError: 'com.fasterxml.jackson.core.json.JsonWriteContext com.fasterxml.jackson.dataformat.cbor.CBORGenerator.getOutputContext()'
com.fasterxml.jackson.dataformat.cbor.CBORGenerator.close(CBORGenerator.java:903)
com.amazonaws.protocol.json.SdkJsonGenerator.close(SdkJsonGenerator.java:267)
com.amazonaws.protocol.json.SdkJsonGenerator.getBytes(SdkJsonGenerator.java:282)
com.amazonaws.protocol.json.SdkCborGenerator.getBytes(SdkCborGenerator.java:67)
	... 13 more
 (Session: 1684968654543585281)`

Currently I'm adding %jvmoption -Dcom.amazonaws.sdk.disableCbor=true;  to all UDFs to make it work:
`--/
CREATE OR REPLACE JAVA SET SCRIPT KINESIS_SCHEMA.KINESIS_METADATA (...) 
EMITS (KINESIS_SHARD_ID VARCHAR(130), SHARD_SEQUENCE_NUMBER VARCHAR(2000)) AS
%jvmoption -Dcom.amazonaws.sdk.disableCbor=true;
%scriptclass com.exasol.cloudetl.kinesis.KinesisShardsMetadataReader;
%jar /buckets/bfsdefault/kinesis/exasol-kinesis-connector-extension-0.1.0.jar;
/
;

--/
CREATE OR REPLACE JAVA SET SCRIPT KINESIS_SCHEMA.KINESIS_IMPORT (...) EMITS (...) AS
%jvmoption -Dcom.amazonaws.sdk.disableCbor=true;
%scriptclass com.exasol.cloudetl.kinesis.KinesisShardDataImporter;
%jar /buckets/bfsdefault/kinesis/exasol-kinesis-connector-extension-0.1.0.jar;
/
;

--/
CREATE OR REPLACE JAVA SET SCRIPT KINESIS_SCHEMA.KINESIS_CONSUMER (...) EMITS (...) AS
%jvmoption -Dcom.amazonaws.sdk.disableCbor=true;
%scriptclass com.exasol.cloudetl.kinesis.KinesisImportQueryGenerator;
%jar /buckets/bfsdefault/kinesis/exasol-kinesis-connector-extension-0.1.0.jar;
/
;`

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #18

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -9,15 +9,15 @@ object Dependencies {
   // Runtime dependencies versions
   private val ExasolVersion = "6.1.7"
   private val ImportExportUDFVersion = "0.1.0"
```
Comment (morazow) [inline project/Dependencies.scala:11]: Could you please also update this? The latest should be should be `0.2.0`.
I think it already includes jackson-databind and jackson-module.

File: doc/changes/changes_0.1.1.md [hunk 2]
```diff
@@ -12,3 +16,10 @@
 * #15: Added a link to the development guide.
 
 ## Dependency updates
+
+* Added `com.fasterxml.jackson.core:jackson-dataformat-cbor:2.12.0`
+* Updated `com.fasterxml.jackson.core:jackson-databind:2.11.2` to `2.12.0`
```
Comment (morazow) [inline doc/changes/changes_0.1.1.md:21]: Use latest aggreed format. 
```
Updated to `artifact:version` (was `previous-version`)
```

File: src/it/scala/com/exasol/cloudetl/kinesis/KinesisAbstractIntegrationTest.scala [hunk 3]
```diff
@@ -18,11 +19,10 @@ trait KinesisAbstractIntegrationTest extends AnyFunSuite with BeforeAndAfterAll
   val TEST_SCHEMA_NAME = "kinesis_schema"
   var assembledJarName: String = _
 
-  val exasolContainer = new ExasolContainer(
-    ExasolContainerConstants.EXASOL_DOCKER_IMAGE_REFERENCE
-  )
+  val exasolContainer = new ExasolContainer("7.0.4")
```
Comment (morazow) [inline src/it/scala/com/exasol/cloudetl/kinesis/KinesisAbstractIntegrationTest.scala:22]: Update travis.yml to download it before tests. There is latest, but good idea to specificly download this version also.
```
before_install:
 - docker pull exasol/docker-db:7.0.3
```

File: src/it/scala/com/exasol/cloudetl/kinesis/KinesisAbstractIntegrationTest.scala [hunk 4]
```diff
@@ -98,7 +98,6 @@ trait KinesisAbstractIntegrationTest extends AnyFunSuite with BeforeAndAfterAll
     statement.execute(
       s"""CREATE OR REPLACE JAVA SET SCRIPT KINESIS_METADATA (...)
          |EMITS (KINESIS_SHARD_ID VARCHAR(130), SHARD_SEQUENCE_NUMBER VARCHAR(2000)) AS
-         |     %jvmoption -Dcom.amazonaws.sdk.disableCbor=true;
```
Comment (morazow) [inline src/it/scala/com/exasol/cloudetl/kinesis/KinesisAbstractIntegrationTest.scala:101]: I thought this should not be removed for integration tests?

**Review by morazow — CHANGES_REQUESTED** [review]

File: doc/changes/changes_0.1.1.md (hunk 2 above)
Comment (AnastasiiaSergienko) [reply doc/changes/changes_0.1.1.md:21]: When was it changed? :O I thought we agreed on this one and this is a template: https://github.com/exasol/release-droid/blob/main/doc/user_guide/changes_file_template.md

File: src/it/scala/com/exasol/cloudetl/kinesis/KinesisAbstractIntegrationTest.scala (hunk 4 above)
Comment (AnastasiiaSergienko) [reply src/it/scala/com/exasol/cloudetl/kinesis/KinesisAbstractIntegrationTest.scala:101]: This one surprisingly works without this option. The data extraction doesn't.

File: doc/changes/changes_0.1.1.md (hunk 2 above)
Comment (AnastasiiaSergienko) [reply doc/changes/changes_0.1.1.md:21]: I like the old format more...

Comment (morazow) [reply doc/changes/changes_0.1.1.md:21]: I think we misunderstood, this is should be default.

**Review by morazow — APPROVED** [review]

**Pull request merged**